READMEs/install.md

```diff
@@ -25,23 +25,40 @@ python3 -m venv venv
 source venv/bin/activate
 ```
 
-##### Install the library
+Now that we have barge running, we can mostly ignore its console while it runs.
```

Comment:
I'm confused. So in this section called "Download barge and run services" you are giving the commands to create a project directory and to activate the venv. But, we say that we have barge running. 
I don't see the download barge and run services commands

Reply:
Good catch! Looks like my commit that merged from main inadvertently caused this issue.

I just pushed a fix.